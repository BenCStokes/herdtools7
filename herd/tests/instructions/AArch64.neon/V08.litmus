AArch64 V08
(* Tests load / store immediate, no offset *)
{
uint8_t x=1; uint8_t y;
0:X0=x; 0:X1=y
}

 P0             ;
 LDR B0, [X0]   ;
 STR B0, [X1]   ;
 forall (y=1)
