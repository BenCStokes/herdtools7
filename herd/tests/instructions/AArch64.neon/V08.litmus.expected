Test V08 Required
States 1
y=1;
Ok
Witnesses
Positive: 1 Negative: 0
Condition forall (y=1)
Observation V08 Always 1 0
Hash=8aad4753b4fbb90fcb565ce80b183e71

